Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -41,15 +41,12 @@ extension UIImageView {
     // This is a helper function for custom async loaders. It starts an operation that will check for the image in
     // a cache (either one passed in or the default if none is specified). If its found in the cache its returned,
     // otherwise, block is run and should return an image to show.
-    private func runBlockIfNotInCache(key: String, var cache: SDImageCache? = nil, completed: CompletionBlock, block: () -> UIImage?) {
+    private func runBlockIfNotInCache(key: String, cache: SDImageCache, completed: CompletionBlock, block: () -> UIImage?) {
```

Comment:
I checked where `runBlockIfNotInCache` is used and made sure the behavior is the same. I thought this made a code a bit more robust because it is now up to the caller to provide a good instance of an `SDImageCache`.